sites/platform/src/dedicated-environments/dedicated-gen-3/overview.md

```diff
@@ -0,0 +1,172 @@
+| **Vault KMS** | 1.12 |
+
+See the [services documentation](https://docs.platform.sh/add-services.html) for service-specific details.
+
+#### Local mounts 
+
+Dedicated Gen 3 provides a redundant infrastructure and local mounts aren’t shared between the three hosts.If you need a folder to be shared between your hosts, set up a [network storage mount](https://docs.platform.sh/add-services/network-storage.html).
+
+### Dedicated Gen 3 vs Dedicated Gen 2
+
+Just like Dedicated Gen 3, [Dedicated Gen 2](dedicated-environments/dedicated-gen-2) ensures increased uptime and availability for your apps and services. But as a Dedicated Gen 2 user, you have to go through the Platform.sh Customer Success team to make configuration or application topology changes.
+
+Dedicated Gen 3 gives you both the high availability of Dedicated Gen 2 and the self-service flexibility and features of Grid projects. As a Dedicated Gen 3 user, you can edit your configuration yourself and see those changes reflected in your environments on every push without opening a ticket. See the table below for more differences and similarities between Dedicated Den 3 and Dedicated Gen 2:
+
+| Feature | Dedicated Gen 2 | Dedicated Gen 3 |
+| --- | --- | --- |
+| **Source Operations** | Yes | Yes |
+| **PHP version upgrade** | Self-service via yaml config files | Self-service via yaml config files |
+| **NodeJS version upgrade** | Self-service via yaml config files | Self-service via yaml config files |
+| **Cron management** | Self-service via yaml config files | Self-service via yaml config files |
+| **Web server internal config : locations**  | Self-service via yaml config files | Self-service via yaml config files |
+| **CDN** | Fastly  | Fastly  |
+| **Dedicated IP** | Yes | Yes |
+| **Usable regions** | Any region needed | Any region needed  |
+| **Autonomous upsize** | Managed through Platform.sh | Managed through Platform.sh|
+| **Multi AZ** | Yes | Yes |
+| **New Relic** | APM + New Relic infrastructure | APM + New Relic infrastructure |
+| **Multi-app support (PWA)** | Supported through docroots | Supported natively |
+| **Routes management**  | Managed by Platform.sh | Self-service via yaml config files |
+| **Environment clone** | Only on development environments | Yes on all branches |
+| **Services : Add, remove, upgrade**  | Managed by Platform.sh | Self-service via yaml config files |
+| **Relationships : Add, remove, update** | Managed by Platform.sh| Self-service via yaml config files |
+| **Mounts management** | - Autonomous or managed by Platform.sh | Self-service via yaml config files |
+| **Workers management** | Managed by Platform.sh | Self-service via yaml config files |
+| **Web server internal config: domains** | Managed by Platform.sh | Self-service via yaml config files |
+| **Storage allocation between mounts, DB and services** | Managed by Platform.sh | Self-service via yaml config files |
+| **Storage increase responsibility** | PSH | Autonomous |
```

Comment:
"PSH" is an internal acronym, and I assume we mean that store requires a support ticket. For consistency with the rest of the table, should this be changed to "Managed by Platform.sh"?